core/src/Serial/Kokkos_Serial.cpp

```diff
@@ -147,7 +170,9 @@ Serial::Serial(NewInstance)
     : m_space_instance(new Impl::SerialInternal, [](Impl::SerialInternal* ptr) {
         ptr->finalize();
         delete ptr;
-      }) {}
+      }) {
+  m_space_instance->initialize();
```

Comment:
So this was an oversight right?  It means that the `m_is_initialized` member was never set to `true` but that somehow didn't cause an error :/

Reply:
Right, before this pull request that only affected ` SerialInternal::is_initialized()`.